src/SSW.SophieBot/SSWSophieBot/language-generation/en-us/agenda.en-us.lg

```diff
@@ -0,0 +1,405 @@
+[Icons](icons.en-us.lg)
+[billableList](billableList.en-us.lg)
+
+# AppointmentAgenda(displayName, appointments)
+- IF: ${count(appointments) > 0}
+    - ```
+    ${AppointmentTable(displayName, appointments)}
+    ```
+- ELSE:
+    - ```
+    ${AppointmentNoData()}
+    ```
+
+# AppointmentNoData()
+- ```
+{
+    "type": "AdaptiveCard",
+    "body": [
+        {
+            "type": "TextBlock",
+            "text": "No calendar available",
+            "wrap": true,
+            "horizontalAlignment": "Center",
+            "weight": "Lighter",
+            "isSubtle": true
+        }
+    ],
+    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+    "version": "1.2"
+}
+```
+
+# AppointmentTable(displayName, appointments)
+- ```
+{
+    "type": "AdaptiveCard",
+    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+    "version": "1.3",
+    "body": [
+        ${MainTitle(concat(displayName, "'s next appointments are"), false)},
```

Comment:
How about changing `Adam Cogan’s next appointments are` to `Adam Cogan's Following Appointments`?

Reply:
The title is exactly same as initially required. Any better phrase can be left to later discussion